docs/bitcoin-stake-expansion.md

```diff
@@ -0,0 +1,617 @@
+   the original staking transaction through the on-demand unbonding
+   script path. The Babylon Genesis chain records the request.
+2. **Covenants Verify the Request**: The Covenants identify
+   the stake expansion request and verify it by submitting
+   their signatures for both the spending of the original
+   staking transaction and the on-demand unbonding and slashing
+   transactions of the expanded staking transaction.
+3. **Submission to Bitcoin**: The staker, having received all
+   necessary signatures submits the fully signed expanded staking
+   transaction to Bitcoin.
+4. **Monitoring for Confirmation**: A vigilante tracker (or the staker themself)
+   monitor Bitcoin for the confirmation status of the stake expansion transaction.
+   Once this happens, the inclusion proof is submitted to Babylon Genesis.
+5. **Stake Expansion Activation**: Once a valid proof of inclusion in a sufficiently confirmed block
+   is submitted for a stake expansion, the expanded staking transaction becomes active
+   and the original staking transaction becomes unbonded.
+   *The switch is an atomic operation*.
+
+> **Note**: The stake expansion procedure is similar to the
+> pre-staking registration procedure in the sense that the
+> staker first submits to Babylon Genesis before broadcasting to
+> Bitcoin.
+
+```mermaid
+sequenceDiagram
+actor btcstaker as BTC Staker
+participant babylon as Babylon Genesis
+participant covd as Covenant
+participant vigilante as Vigilante
+participant btc as BTC
+
+autonumber
+
+Note over btcstaker,babylon: ...Active BTC delegation
+btcstaker->>babylon: MsgBtcStakeExpand
+activate babylon
+babylon-->>btcstaker: New pending BTC Del
+deactivate babylon
+
+Note over covd,babylon: Watch for pending BTC Dels
+
+covd->>babylon: Send missing signatures
+activate babylon
+babylon-->>covd: Stake Expansion Verified
+deactivate babylon
+
+Note over vigilante,btc: Watch BTC
+
+btcstaker->>btc: Send Stake Expanion BTC Tx (spends the staking output)
+activate btc
+btc-->>btcstaker: Stake Expansion tx mined
+deactivate btc
+
+
+Note over btcstaker, btc: Stake expansion is K deep
+vigilante->>babylon: Reports BTC stake expansion is K deep<br>MsgBTCUndelegate
+activate babylon
+babylon-->>vigilante: Stake expansion is confirmed
+deactivate babylon
+Note over babylon: Creates events of unbonding for old staking<br>and activation of stake expansion to process<br>at the same babylon block height
+```
+
+### 2.2. Stake Expansion Transaction Structure
+
+All expanded staking transactions are normal
+Bitcoin staking transaction as per the
+[specification](./staking-script.md#bitcoin-staking-transactions-specification).
+What makes them unique is the requirement that they follow the following structure:
+* **Two Inputs** in the following precise order:
+  * **First Input**: The staking output of the original staking transaction.
+    Its value should hold at least as many BTC as the original staking output. 
```

Comment:
This sentence is not needed. Inputs in BTC are pointers to previous outputs and they not hold any value by themselves. 

I think this sentence should be mover the:
> * **Any number of outputs** with at least *one* corresponding to the
  expanded staking transaction staking output.

paragraph, as it is here this requirement becomes something custom. Babylon chain requires that the new staking output value must be at least the same as the ols taking value